docs/source/install/upgrades.rst

```diff
@@ -217,6 +221,14 @@ v3.4
     # ensure python3.6 package exists and could be installed
     apt-cache show python3.6
 
+*  *Ubuntu 16 and RHEL/CentOS 7 only*. All packs installed prior to upgrade will need to have their virtual environment re-created after upgrading |st2| packages (on all nodes which run st2actionrunner services), using the following commands:
+
+.. sourcecode:: bash
+
+    sudo rm -r /opt/stackstorm/virtualenvs
```

Comment:
Do we also need ``-f`` flag?

Reply:
Also, I guess this will also delete whole ``virtualenvs`` directory, not sure if that's a problem or not.

Perhaps ``rm -rf /opt/stackstorm/virtualenvs/*`` is better.

Reply:
It worked on my environment without the -f, but I've updated to the version you mentioned, as agree it's better, and no harm in using the -f in case there are files in some environments that need it.